cli/command/manifest/cmd.go

```diff
@@ -0,0 +1,45 @@
+package manifest
+
+import (
+	"fmt"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+
+	"github.com/spf13/cobra"
+)
+
+// NewManifestCommand returns a cobra command for `manifest` subcommands
+// nolint: interfacer
+func NewManifestCommand(dockerCli *command.DockerCli) *cobra.Command {
+	// use dockerCli as command.Cli
```

Comment:
The `nolint` line and `use dockercli as command.Cli` line can be removed here by changing `*command.DockerCli` to `command.Cli`